src/components/value-list/resources.ts

```diff
@@ -8,3 +8,7 @@ export enum ICON_TYPES {
 export const TEXT = {
   filterResults: "Filter results"
 };
+
+export const HANDLE_LABEL = "press space and use arrow keys to reorder content";
```

Comment:
I think we need an `intl` property to go along with this so that users can localize it.